internal/internal_task_handlers_test.go

```diff
@@ -389,6 +390,8 @@ func (t *TaskHandlersTestSuite) TestWorkflowTask_NondeterministicDetection() {
 		NonDeterministicWorkflowPolicy: NonDeterministicWorkflowPolicyBlockWorkflow,
 	}
 	taskHandler := newWorkflowTaskHandler(testDomain, params, nil, getHostEnvironment())
+	newWorkflowTaskWorkerInternal(taskHandler, t.service,testDomain, params)
```

Comment:
Will this do something? Could you add a unit test for your change?

Reply:
This will set the laTunnel for the taskHandler, without it, the tests will end up panic as taskHandler.laTunnel is nil 

I will try to see if I can add some unit tests for it.